src/website/frontend/styles/lib/variables.scss

```
@@ -0,0 +1,33 @@
+$mainBreakPoint: 580px;
+
+$mainPadding: 160px;
```

Stylelint (17.6.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)